addons/miscFixes/config.cpp

```diff
@@ -82,3 +82,11 @@ class CfgWeapons {
         };
     };
 };
+
+class CfgInGameUI {
+    class SquadRadar {
+        //78-64-40
+        // backgroundTexture = "";
+        compassBackgroundTexture = QUOTE(PATHTOF(ui\squadRadarBackground_Darker_ca.paa));
```

Comment:
In vanilla it's a texture with the 3 rings and compass dirs.  I just filled in the inside area